downstream/modules/platform/proc-controller-create-notification-template.adoc

```diff
@@ -0,0 +1,15 @@
+[id="controller-create-notification-template"]
+
+= Creating a notification template
+
```

Comment:
Use the following procedure to create a Notification template:  or some other introduction.